dev/tasks/conda-recipes/r-arrow/build.sh

```diff
@@ -1,9 +1,14 @@
-#!/usr/bin/env bash
-
+#!/bin/bash
 set -ex
 
 export DISABLE_AUTOBREW=1
-# See https://conda-forge.org/docs/maintainer/knowledge_base.html#newer-c-features-with-old-sdk
-export ARROW_R_CXXFLAGS="${ARROW_R_CXXFLAGS} -D_LIBCPP_DISABLE_AVAILABILITY"
 
-$R CMD INSTALL --build r/.
+# set C++17 due to abseil
+export ARROW_R_CXXFLAGS="${ARROW_R_CXXFLAGS} -std=c++17"
+
+if [[ "${target_platform}" == osx-* ]]; then
+    # See https://conda-forge.org/docs/maintainer/knowledge_base.html#newer-c-features-with-old-sdk
+    export ARROW_R_CXXFLAGS="${ARROW_R_CXXFLAGS} -D_LIBCPP_DISABLE_AVAILABILITY"
+fi
+
+${R} CMD INSTALL --build r/. ${R_ARGS}
```

Comment:
`${R_ARGS}` is apparently necessary to be able to cross-compile

Reply:
Could you add this note as comment in this file?